core/src/storage.rs

```diff
@@ -22,12 +22,21 @@ impl LocalStorage {
             .join(DATA_DIR);
         for path in &[&temp_dir, &data_dir] {
             if !path.exists() {
-                fs::create_dir(&path)?;
+                fs::create_dir(path)?;
             }
         }
         Ok(Self { temp_dir, data_dir })
     }
 
+    pub fn remove_version(&self, release_version: &str) -> Result<()> {
```

Comment:
Can you incorporate this function for the CLI? There is already an `uninstall` function and you can call `remove_version` there.

https://github.com/spicylobstergames/SpicyLauncher/blob/267559381422c0a23611d50ea47b09cd2b3fa8cc/cli/src/app.rs#L131

Reply:
Updated it to use the `remove_version` function, and to make sure the `exists()` check was consistent with the `remove_version` function I added a `version_path()` helper.